core/charm/repository/charmhub_test.go

```diff
@@ -818,14 +818,14 @@ func (selectNextBaseSuite) TestSelectNextBaseWithCentosBase(c *gc.C) {
 		Platform: corecharm.Platform{
 			Architecture: "amd64",
 			OS:           "ubuntu",
-			Series:       "focal",
+			Channel:      "20.04",
 		},
 	})
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(platform, gc.DeepEquals, []corecharm.Platform{{
 		Architecture: "amd64",
 		OS:           "centos",
-		Series:       "centos7",
+		Channel:      "centos7",
```

Comment:
Hopefully we can fix this at some point.

Reply:
yeah, will need a separate PR and consideration for updating state etc. really want to do it